recipes/oommf/meta.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: oommf
+  version: "1.2b0_20160930b1"  # [unix]
+  version: "1.2b0_20160930b0"  # [win]
+
+# on Windows, we install from precompiled binary packages
+source:  # [unix]
+  git_url: https://github.com/fangohr/oommf  # [unix]
+  git_rev: 1.2b0_20160930b1  # [unix]
+
+build:
+  number: 0
+  skip: True  # [win32]
+
+requirements:
+  build:
+    - tk 8.5*  # [unix]
+    - tk 8.6*  # [win]
+    - curl  # [win]
+    - 7za  # [win]
+  run:
+    - tk 8.5*  # [unix]
+    - tk 8.6*  # [win]
+
+test:  # [unix]
+  commands:  # [unix]
+    - oommf +version  # [unix]
```

Comment:
The `oommf` command line interface requires `tclsh`, which doesn't seem to be there on Windows in the conda packages of tcl/tk. For our Python wrapper code which calls OOMMF, we've found a way of [calling what we need](https://github.com/joommf/oommfc/blob/943ddc0476fde2f8cc34629687636a569fccbc4f/oommfc/oommf.py#L117) without the `oommf` command.